packages/bpk-component-popover/stories.scss

```diff
@@ -3,3 +3,11 @@
 .bpk-popover-paragraph {
   margin-bottom: $bpk-spacing-xs;
 }
+
+.bpk-popover-spacer {
+  display: flex;
+  height: 500 $bpk-one-pixel-rem;
```

Comment:
Should there be a multiplication symbol here?

Reply:
Good spot!